tests/CoreTests/src/algebra.cpp

```diff
@@ -36,6 +38,70 @@ void run() {
                "Twist should be around z" );
     RA_VERIFY( Ra::Core::AngleAxis( qs ).axis().dot( Ra::Core::Vector3::UnitZ() ) == 0,
                "Swing should be in xy" );
+
+    //    using std::clamp;
+    //    using Ra::Core::Math::clamp;
+    Scalar min = -12.;
+    Scalar max = +27.;
+    Scalar s = 0.6;
+
+    RA_VERIFY( s == std::clamp( s, min, max ), "Clamp fails on Scalar" );
+    RA_VERIFY( min == std::clamp( min, min, max ), "Clamp fails on Scalar" );
+    RA_VERIFY( max == std::clamp( max, min, max ), "Clamp fails on Scalar" );
+    RA_VERIFY( max == std::clamp( max + s, min, max ), "Clamp fails on Scalar" );
+    RA_VERIFY( min == std::clamp( min - s, min, max ), "Clamp fails on Scalar" );
```

Comment:
do we really need to test the std version?

Reply:
do not cost so much, so yes ;)